### ISSUE TITLE
Missing reference for 'GuzzleHttp\Stream\BufferStream'

### ISSUE THREAD

**Issue by jbryu** [Understeam/yii2-jira#9]
When I look at `understeam\jira\Client`, `GuzzleHttp\Stream\BufferStream` is being used without a Composer package required. I think `guzzlehttp/streams` wasn't added to composer.json. I don't know what version you prefer to have here, but `~3.0.0` works for me. Does `guzzlehttp/streams` suppose to be part of `understeam/yii2-httpclient`?

```
"require": {
    ...
    "guzzlehttp/streams": "~3.0.0",
    "understeam/yii2-jira": "~0.3",
    ...
},
```

Thank you,
Jongbeom

**Comment by AnatolyRugalev** [comment]
Good catch. Looks like you're right. I thought this package was required by guzzle library.

Feel free to make a PR for this.
